app/Livewire/Server/CloudProviderToken/Show.php

```
@@ -0,0 +1,144 @@
+
+    public $parameters = [];
+
+    public function mount(string $server_uuid)
+    {
+        try {
+            $this->server = Server::ownedByCurrentTeam()->whereUuid($server_uuid)->firstOrFail();
+            $this->loadTokens();
+        } catch (\Throwable $e) {
+            return handleError($e, $this);
+        }
+    }
```

PHPMD (2.15.0)

The parameter $server_uuid is not named in camelCase. (undefined)

(CamelCaseParameterName)